### PR TITLE
misc hint'ish cleanup

### PR DESCRIPTION
**Pull request by altendky**

### CODE REVIEW

File: .github/workflows/upload-pypi-source.yml [hunk 1]
```diff
@@ -49,6 +49,14 @@ jobs:
         python -m pip install --upgrade pip setuptools wheel
         python -m pip install .[dev]
 
+    - name: Lint source with flake8
```
Comment (Yostra) [inline .github/workflows/upload-pypi-source.yml:52]: do we need these here? They are ran by superlinter/pre-commit already

Comment (altendky) [reply .github/workflows/upload-pypi-source.yml:52]: Shouldn't that be a separate discussion from PAC?

Comment (altendky) [reply .github/workflows/upload-pypi-source.yml:52]: And specifically, mypy is run most correctly here.

Comment (Yostra) [reply .github/workflows/upload-pypi-source.yml:52]: I guess, lets merge this

**Pull request merged**